lib/atom-css-sniffer.js

```diff
@@ -4,16 +4,96 @@ let helpers;
 let path;
 let filesystem;
 
+// Caches
+let versions = {};
+let cmds = {}
+
 function loadDeps() {
-  if (!helpers) {
-    helpers = require('atom-linter');
-  }
-  if (!path) {
-    path = require('path');
-  }
-  if (!filesystem) {
-    filesystem = require('fs');
-  }
+    if (!helpers) {
+        helpers = require('atom-linter');
+    }
+    if (!path) {
+        path = require('path');
+    }
+    if (!filesystem) {
+        filesystem = require('fs');
+    }
+}
+
+function _getCommand(projectPath) {
```

Comment:
- project_path
- Do we add JSDoc to functions?

Reply:
I think we try to follow the [angular style rules](https://angular.io/guide/styleguide#coding-conventions). However as long as there is no sniffer it's up in the air tbh.